### PR TITLE
Adds config option to disable colored output

### PR DESCRIPTION
**Pull request by jforcada**
Hi!

This pull request is to be able to deactivate the colored output from the logs. We found very hard to read raw logs from the current output, and also it is pretty hard to search and filter them in tools like kibana.

Hope you like it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ Creates a new GoodConsole object with the following arguments:
 - `[config]` - optional configuration object with the following available keys
 	- `format` - [MomentJS](http://momentjs.com/docs/#/displaying/format/) format string. Defaults to 'YYMMDD/HHmmss.SSS'.
 	- `utc` - boolean controlling Moment using [utc mode](http://momentjs.com/docs/#/parsing/utc/) or not. Defaults to `true`.
+	- `color` - boolean activating the colored output or not. Defaults to `true`.
```
Comment (AdriVanHoudt) [inline README.md]: I would personally phrase it as `a boolean specifying whether to output in color` (like in the hapi docs)

**Comment by jforcada** [comment]
Yes, that's simpler. Corrected.

**Comment by AdriVanHoudt** [comment]
:+1:

**Comment by Marsup** [comment]
@arb is this thing going somewhere ?

**Pull request merged**